stablehlo/dialect/AssemblyFormat.cpp

```diff
@@ -276,6 +278,73 @@ ParseResult parseDenseI64Array(OpAsmParser& parser,
   return success();
 }
 
+void printSliceRanges(OpAsmPrinter& p, Operation* op,
+                      DenseIntElementsAttr start_indices,
+                      DenseIntElementsAttr limit_indices,
+                      DenseIntElementsAttr strides) {
```

Comment:
This function still appears to be using the Google naming style for its parameters. (Yes, we should really set up an automatic check for this kind of stuff - we tried clang-tidy at some point and it turned out to be too slow, but that was before we got more powerful runners).